### DISCUSSION TITLE
How to use Icedrive ?

### DISCUSSION THREAD

**Discussion by dianedef** [icidasset/diffuse#400] [Q&A] [answered]
Please add it as a source if not possible ? Thank you for the amazing work on Diffuse 🙏

**Comment by icidasset** [comment]
Thanks ☺️ It looks like Icedrive doesn’t have a public API yet so I can’t add it as a separate source type. But.. they do support WebDAV you could see if that works with Diffuse: 
https://icedrive.net/help/account/does-icedrive-support-webdav

Let me know if that works for you.
WebDAV is quite unreliable so 🤞

**Reply by dianedef** [reply]
I generated an auth key and all but I got the message "cannot connect to this source', and no more info 🤷‍♂️🤷‍♂️😵‍💫

**Reply by icidasset** [reply, marked as answer]
Too bad 😞 Unfortunately I can't test it because it requires a paid account to use the WebDAV feature. If you do want to investigate, there should be some more information in the developer console. It's most likely because they don't support CORS, usually isn't considered because WebDAV is usually not used this way.
